src/routes/safes/safes.service.ts

```diff
@@ -150,6 +157,26 @@ export class SafesService {
     }
   }
 
+  private async modifiedMessageTag(
+    chainId: string,
+    safeAddress: string,
+  ): Promise<Date | null> {
+    const messages = await this.messagesRepository.getMessagesBySafe(
+      chainId,
+      safeAddress,
+    );
+
+    if (messages.results.length === 0) {
+      return null;
+    }
+
+    const sortedMessages = messages.results.sort((m1, m2) => {
+      return m2.modified > m1.modified ? 1 : -1;
```

Comment:
nitpick: we can set the `compareFn` as the difference between the dates.

```typescript
(m1, m2) => {
  return m2.modified.getTime() - m1.modified.getTime();
}
```

It should return 0 if the dates are equal [^1] (although it works for this specific scenario).

[^1]: https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/array/sort#syntax

Reply:
Much cleaner! I've adjusted it accordingly.